tpot/base.py

```diff
@@ -501,6 +500,35 @@ def _fit_init(self):
             self._last_optimized_pareto_front_n_gens = 0
             self._setup_config(self.config_dict)
 
+            if multi_output_target:
```

Comment:
Modifying _config_dict may not work in the situation that use use a customized configurations  instead of default one. So, I think a practical way is to modify the `_compile_to_sklearn` function ([here](https://github.com/EpistasisLab/tpot/blob/v0.11.1/tpot/base.py#L1193-L1194)). If `multi_output_target` is `True`, then
`sklearn_pipeline=MultiOutputClassifier(estimator=sklearn_pipeline)` or  `sklearn_pipeline=MultiOutputRegessor(estimator=sklearn_pipeline)` . I think it maybe a more general solution for multioutput dataset.

Reply:
This seems better, to be honest i didn't find a good place where to put my code and only settled on the _fit_init function therefore.

Reply:
Okay i looked into it, but the code would be a mess. Several functions would have to take multi_output_target as a new argument (most of them in export_utils.py), since they don't have access to the data or the TPOT Object. 

Imo _fit_init seems to be the least intrusive point to include the checks

Reply:
Thank you for looking into. You are right. I think TPOT exported codes should also include MultiOutputRegessor/MultiOutputClassifier, which should change a lot of codes in TPOT. I will look into it when I get some time next week.

Reply:
any updates?

Reply:
@jhmenke I am sorry for overlooking this. I did not get a chance to look into this issue those days due to my busy schedule. I agree that TPOT need some major changes for including `MultiOutputRegessor/MultiOutputClassifier`. I may get some time in March to add those changes. You are welcome to push any changes meanwhile.

Reply:
Could we use my PR as a temporary fix until you have time to thoroughly refactor the code? I can prepare an update with the current development branch.

Reply:
Okay i merged the current master/development into this. Should be good to go as an interim solution then.